src/main/scala/org/scalastyle/Main.scala

```diff
@@ -47,6 +48,8 @@ object Main {
     println("     --inputEncoding STRING      encoding for the source files")
     println(" -w, --warnings true|false       fail if there are warnings")
     println(" -e, --externalJar FILE          jar containing custom rules")
+    println(" -x, --excludedFiles STRING      regex matches file paths to exclude")
```

Comment:
Looking at the implementation, the doc is a bit terse. You actually accept a list of regexes, separated by `;`.